client/src/lib/query-keys.ts

```diff
@@ -34,9 +37,22 @@ export const projectKeys = createQueryKeys("projects", {
   countries: null,
 });
 
+export const scorecardFiltersKeys = createQueryKeys("scorecardFilters", {
+  all: (
+    tableView: (typeof TABLE_VIEWS)[number],
+    filters?: z.infer<typeof scorecardFiltersSchema> & {
+      sorting?: SortingState;
+      pagination?: PaginationState;
+    },
+  ) => ["all", tableView, filters],
+  id: (id: string) => [id],
+  countries: null,
+});
+
```

Comment:
Can we merge this and `projectKeys` so we a more generic factory key for the tables? I think having a more generic name and using a generic for the filters should be enough.